src/components/AllMovies.js

```diff
@@ -27,6 +27,7 @@ class AllMovies extends React.Component {
       <div className="section--all-movies">
         <section className="heading--all-movies">
           <h2>Movies</h2>
+          <h2>{ this.state.error }</h2>
```

Comment:
This H2 on line 30 is going to be an empty string most of the time which could be really confusing to anyone using a screen reader. Better to only show this if the Error is there. Remember that && Syntax? Might be good to refactor :)